plugins/irma-web/translations/en.js

```diff
@@ -1,18 +1,19 @@
+// prettier-ignore
```

Comment:
For readability of the locales files, I disabled the formatter for these files to prevent it from enforcing line breaks.